extensions/amp-mustache/0.1/test/test-amp-mustache.js

```diff
@@ -262,108 +293,139 @@ describe('amp-mustache 0.1', () => {
           'text after a template');
     });
 
-    it('should sanitize nested templates without type="amp-mustache"', () => {
-      templateElement./*OK*/innerHTML =
-          'text before a template ' +
-          '<template>text inside template</template> ' +
-          'text after a template';
-      template.compileCallback();
-      const result = template.render({});
-      expect(result./*OK*/innerHTML).to.equal(
-          'text before a template  text after a template');
-    });
-
-    it('should not render variables inside a nested template', () => {
-      templateElement./*OK*/innerHTML =
-          'outer: {{outerOnlyValue}} {{mutualValue}} ' +
-          '<template type="amp-mustache">nested: {{nestedOnlyValue}}' +
-          ' {{mutualValue}}</template>';
-      template.compileCallback();
-      const result = template.render({
-        outerOnlyValue: 'Outer',
-        mutualValue: 'Mutual',
-        nestedOnlyValue: 'Nested',
-      });
-      expect(result./*OK*/innerHTML).to.equal(
-          'outer: Outer Mutual ' +
-          '<template type="amp-mustache">nested: {{nestedOnlyValue}}' +
-          ' {{mutualValue}}</template>');
-    });
+    it('should sanitize nested templates without type="amp-mustache"',
+        done => {
+          if (isTemplateType) {
+            innerHtmlSetup(
+                'text before a template ' +
+                '<template>text inside template</template> ' +
+                'text after a template');
+            template.compileCallback();
+            const result = template.render({});
+            expect(result./*OK*/innerHTML).to.equal(
+                'text before a template  text after a template');
+            done();
+          } else {
+            done();
```

Comment:
Passing and calling `done` shouldn't be necessary here. Also, extract the `isTemplateType` conditional:

```js
if (isTemplateType) {
  describe('with template[type=amp-mustache]') {
    it(...);
    it(...);
  }
}
```

Reply:
the test would hang otherwise on the the condition script[type=text/plain] [template=amp-mustache]. But extracting the condition outside and not running it at all does remove the need for done(), if that's you are saying. Thanks